canarytokens/constants.py

```diff
@@ -25,3 +25,6 @@
 CANARYTOKEN_LENGTH = 25  # equivalent to 128-bit id
 
 CANARY_PDF_TEMPLATE_OFFSET: int = 793
+
+# Maximum number of alert failures before a mechanism is disabled
+MAX_ALERT_FAILURES = 5
```

Comment:
Do we not want to maybe make this configurable? With the default being 5.

Reply:
Good call, moved to Settings.